Sources/ContextPropagation/Context.swift

```diff
@@ -0,0 +1,29 @@
+public protocol ContextKey {
+    associatedtype Value
+}
+
+public struct Context {
```

Comment:
Do we want to leave out the naming bikeshedding for now or tackle it right away? :-)

(Long story short, I'm a bit worrying about the "catch all" name that context is and keen on phrasing it as baggage since we "carry it around" with inspiration from terminology https://github.com/tracingplane/tracingplane-go/blob/master/tracingplane/baggagecontext.go and the in the paper we've been reviewing and getting inspiration from 🤔) Let's have the naming as a separate topic though, ignore for now! 🙇

Reply:
I agree that `Context` is a bit too generic. I'd say we can leave it like this as long as we are the only users of the API and can simply Refactor>Rename once we found a better name for this informed by the different use cases.

Reply:
I'm ok with that :) Let's revisit at some point